rest-api/etl/ddl.sql

```diff
@@ -456,10 +461,18 @@ CREATE TABLE measurement
     unit_source_value varchar(50),
     value_source_value varchar(50),
     meas_id bigint,
+    parent_id bigint,
```

Comment:
This PR introduces `parent_id` but doesn't use it?

Reply:
It is implicitly used when inserting into the measurement table. (Without it the insert fails.)